tests/test_login.py

```diff
@@ -1,10 +1,15 @@
 from fastapi.testclient import TestClient
 from app.main import app
+from tests import utils
 
 client = TestClient(app)
 
-
 def test_login():
+    utils.create_test_user()
     res = client.post("/login", data={"username": "Test", "password": "1234"})
+    utils.delete_test_user()
     assert res.status_code == 200
     assert res.json().get("access_token")
```

Comment:
You are just checking if there a token - specifically you should check if that token is valid and expected one

Reply:
You probably need to think of how to mock that token. 
That's the place where you need to investigate how mocks works